### PR TITLE
notate the lease endpoint reconciler is alpha

### PR DESCRIPTION
**Pull request by rphillips**
Notates the lease endpoint reconciler is alpha

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7305)

### CODE REVIEW

**Comment by ericchiang** [comment]
Ref https://github.com/kubernetes/kubernetes/issues/57617

Tech review lgtm

**Comment by ericchiang** [comment]
@tengqm mind if we merge as is? I don't think this is a big enough change to require an approver.

File: docs/admin/high-availability/building.md [hunk 1]
```diff
@@ -209,14 +209,16 @@ continuously try to add itself to the list of endpoints while removing
...
-Starting with Kubernetes 1.9, a new reconciler implementation is available.
+Starting with Kubernetes 1.9, a new alpha reconciler implementation is available.
 It uses a *lease* that is regularly renewed by each apiserver
 replica. When a replica is down, it stops renewing its lease, and
 the other replicas notice that the lease expired and remove it
 from the list of endpoints. You can switch to the new reconciler
 by adding the flag `--endpoint-reconciler-type=lease` when starting
 your apiserver replicas.
 
+Note: The lease reconciler is in alpha.
```
Comment (heckj) [inline docs/admin/high-availability/building.md]: appreciate referencing this as alpha, but could we use https://kubernetes.io/docs/home/contribute/includes/#feature-state to indicate it?

Comment (rphillips) [reply docs/admin/high-availability/building.md]: can the alpha include be included within a sub-section of a document? From what I can tell it applies to the entire section.

**Comment by jberkus** [comment]
it is now code slush, so this PR needs status/approved-for-milestone to merge.  please label and merge, thanks!

**Comment by rphillips** [comment]
@heckj added the include

**Comment by heckj** [comment]
Thanks @rphillips - just chtted with @jberkus as well, think this is ready to go for an immediate update, so approving to merge it through and finish this out

/approve

**Pull request merged**